src/RestEaseClientGenerator/Builders/ModelBuilder.cs

```diff
@@ -85,7 +85,10 @@ public string Build(RestEaseModel restEaseModel, bool isFirst, bool isLast)
                 builder.AppendLine("        /// </summary>");
             }
 
-            var safePropertyName = property.Name.ToValidIdentifier();
+            var safePropertyName = !string.IsNullOrEmpty(property.Name)
+                ? property.Name.ToValidIdentifier()
```

Comment:
Would it make sense to change the implementation from `ToValidIdentifier` to also allow null string as input, and then return string.Empty?

Or will this have other effects for the rest of the code?

Reply:
Yes I think that's better. I'll try to improve it tomorrow.

Reply:
I've changed the way ToValidIdentifier works and added an early return, making its input a nullable string.